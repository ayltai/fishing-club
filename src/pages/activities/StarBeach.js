// @flow
'use strict';

import React from 'react';
import muiThemeable from 'material-ui/styles/muiThemeable';

const StarBeach = () : any => {
    return (
        <div>Star Beach</div>
    );
};

export default muiThemeable()(StarBeach);
