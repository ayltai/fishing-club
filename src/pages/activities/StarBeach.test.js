'use strict';

import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import StarBeach from './StarBeach';

it('renders correctly', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <BrowserRouter>
            <MuiThemeProvider>
                <StarBeach />
            </MuiThemeProvider>
        </BrowserRouter>,
        div);

    expect(div.innerHTML).toMatchSnapshot();
});
