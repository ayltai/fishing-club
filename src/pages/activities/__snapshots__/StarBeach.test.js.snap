// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `

<div data-reactroot>
  <div class="starBeachItem"
       style="color: rgba(0, 0, 0, 0.87); background-color: rgb(255, 255, 255); box-sizing: border-box; font-family: Roboto, sans-serif; box-shadow: 0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12); border-radius: 2px; z-index: 1;"
  >
    <div style="padding-bottom: 8px;">
      <div style="position: relative;">
        <div>
          <img src="star-beach.jpg"
               style="vertical-align: top; max-width: 100%; min-width: 100%; width: 100%;"
          >
        </div>
        <!-- react-text: 7 -->
        <!-- /react-text -->
      </div>
      <div style="padding: 16px; position: relative;">
        <span style="font-size: 24px; color: rgba(0, 0, 0, 0.87); display: block; line-height: 36px;">
          Star beach (星星灘)
        </span>
        <span style="font-size: 14px; color: rgba(0, 0, 0, 0.54); display: block;">
          A good place for selfies
        </span>
      </div>
      <div style="padding: 16px; font-size: 14px; color: rgba(0, 0, 0, 0.87);">
        <p>
          It is a beach covered entirely by water and only appears when the tide is low. It was a place with no name, and people start to call it "Star beach" because the water there is so clear that many starfishes live in there.
        </p>
        <div style="text-align: center;">
          <img src="starfish.jpg"
               style="max-width: 100%;"
          >
        </div>
        <p>
          Sadly, due to heavy pollutions, we can no longer find that many starfishes there as shown in the above image. But still, we will find some, and you can even put it in your hands to feel when it crawls.
        </p>
      </div>
    </div>
  </div>
  <div class="starBeachItem"
       style="color: rgba(0, 0, 0, 0.87); background-color: rgb(255, 255, 255); box-sizing: border-box; font-family: Roboto, sans-serif; box-shadow: 0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12); border-radius: 2px; z-index: 1;"
  >
    <div style="padding-bottom: 8px;">
      <div style="padding: 16px; position: relative;">
        <span style="font-size: 24px; color: rgba(0, 0, 0, 0.87); display: block; line-height: 36px;">
          How and when to visit Star beach?
        </span>
        <span style="font-size: 14px; color: rgba(0, 0, 0, 0.54); display: block;">
        </span>
      </div>
      <div style="padding: 16px; font-size: 14px; color: rgba(0, 0, 0, 0.87);">
        <p>
          Star beach is covered by water when the tide is high, so we have to wait until the tide is low enough. For only once a day, the tide is at its lowest level. The time of a low tide is not fixed and is shifted about 1 hour each day. That's why we picked certain weekends when a low tide will appear in day time. On that day it will be around 3pm to 5pm, the beach will appear for about 1 hour.
        </p>
        <p>
          The raft owner will pay close attention to the tide and let us know when the time has come. From the raft, we will take a boat and travel to the beach in a few minutes. The beach will be surrounded by shallow water where the boat will not go too close to it, or it will be stranded. Therefore, we will leave the boat, jump into water, and walk to the beach. The water level there is about 2 to 3 feet (about as high as our hip).
        </p>
      </div>
    </div>
  </div>
</div>

`;
